README.md

```diff
@@ -8,283 +8,10 @@ CategoricalArrays.jl
 [![Coveralls](https://coveralls.io/repos/github/JuliaData/CategoricalArrays.jl/badge.svg)](https://coveralls.io/github/JuliaData/CategoricalArrays.jl)
 [![Codecov](https://codecov.io/gh/JuliaData/CategoricalArrays.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/JuliaData/CategoricalArrays.jl)
 
-Tools for working with categorical variables, both with unordered (nominal variables)
```

Comment:
I would keep this paragraph so that people can easily know what this is about (even if it's duplicated in the manual).